src/service.ts

```diff
@@ -197,18 +198,17 @@ export class Service {
     return annotations;
   }
 
-  static async compileFiles(files: File[], from: Language, to: Language, options = ""): Promise<{ [name: string]: (string|ArrayBuffer); }> {
+  static async compileFiles(files: Array<any>, from: Language, to: Language, options = ""): Promise<{ [name: string]: (string|ArrayBuffer); }> {
```

Comment:
Not sure why we are changing File[] to File[][] ?

Reply:
This is all happen just because of  I am not using the service.compileFiles  I made the changes based on the service.compileFile.